.gitignore

```diff
@@ -100,4 +100,10 @@ spark-warehouse/
 metastore_db/
 
 #MacOS
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Swagger API documentation generated on build
+rest-gateway/src/main/resources/static/producerSwagger.json
```

Comment:
Suggestion: instead of putting the generated output into the `src/` and excluding it explicitly here, you can write it into a separate directory under `target/` (that is already excluded from VCS) and attach it to the project source dir list, or web resources list. This is a common practice in handling generated stuff in Maven.

Reply:
Generating direcly into src/main/resources/* let me serve the static html files easilly with spring web according to the reference : https://spring.io/blog/2013/12/19/serving-static-web-content-with-spring-boot

For info, I've tried by curiosity to generate these files outside of ./src/resources/* but instead in ./target/ and it stoped working. 
In the DocumentationConfig.addResourceHandlers, I of course added `registry.addResourceLocations("./target/")`